### PR TITLE
update licensing information for 8.6

### PR DESCRIPTION
**Pull request by conceptualshark**
## Description

Resolves https://github.com/camunda/camunda-docs/issues/4356

Licensing has been updated for 8.6. This PR updates the 8.6 license page according to the [internal draft](https://docs.google.com/document/d/14GbgxWseQ4-5iIuGCWIbq2JpulHp4OBozSe169Fwzzg/edit?usp=sharing). 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by conceptualshark** [comment]
Tagging @camunda/tech-writers for the review on this, but as most of the copy may be specific to legal requirements, I'm not sure how much we have the freedom to update.

File: docs/reference/licenses.md [hunk 1]
```diff
@@ -6,31 +6,29 @@ description: "Licensing information for components of Camunda 8"
...
 
-Operate, Tasklist, Identity, and Optimize are licensed to our users and customers under a proprietary license. The proprietary license can be either the [Camunda Self-Managed Free Edition license](https://legal.camunda.com/licensing-and-other-legal-terms#legal-terms-camunda-platform-c8-self-managed-free-edition-and-beta-offerings) or the Camunda Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda). Note that while the Camunda Self-Managed Free Edition is indeed free, it only allows usage of the software in non-production environments. To use the software in production, [purchase the Camunda Self-Managed Enterprise Edition](https://camunda.com/platform/camunda-platform-enterprise-contact/). This licensing information also applies to web apps and APIs.
+### Camunda Connectors
```
Comment (akeller) [inline docs/reference/licenses.md:17]: Do we need to qualify Connectors as "Camunda Connectors" anywhere else? This stood out to me.

**Comment by conceptualshark** [comment]
Some additional changes will be coming in on the source doc, so waiting until that is in a final state before incorporating any additional edits.

**Comment by conceptualshark** [comment]
Got the okay to move ahead with the current version and update any changes as they come in - re-requesting @camunda/tech-writers review while @akeller is out, and @theburi for visibility.

**Review by mesellings — DISMISSED** [review]
Lgtm 👍  (just one non-blocking suggestion feel fee to ignore)

**Comment by conceptualshark** [comment]
@mesellings implemented your feedback - one more quick review here? Thanks!

**Review by mesellings — APPROVED** [review]
Lgtm 👍

File: docs/reference/licenses.md (hunk 1 above)
Comment (theburi) [reply docs/reference/licenses.md:17]: I don't think we need to qualify, as we are talking about Camudna here in general.

**Pull request merged**